test/e2e/ui-properties/randplanetBtn

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by ruxit on 5/19/16.
+ */
+
+module.exports = {
```

Comment:
Hrm. Name, rotation period, orbital period all have the same css selector -- .mdl-textfield__input. If this was Java I could get them all and pick and choose what I wanted by index. (Name would be get(0). Rotation period, get(1), etc.) ... let me see if there is something similar in JavaScript.